pkg/template/cloudconfig_test.go

```diff
@@ -55,8 +58,17 @@ func TestCloudConfig(t *testing.T) {
 		if err != nil {
 			t.Errorf("failed to render ignition files, %v:", err)
 		}
+		tc.params.Extension = nopExtension{}
 		tc.params.Files = files
-		tc.params.Images = BuildImages("docker.io", Versions{})
+		tc.params.Versions = Versions{
```

Comment:
I don't really like arbitrarily defining these versions in the middle of the test, but it will be clear enough when these need to be updated if we change constraints and this test starts to fail I guess.